src/components/dialogs/create-filter-dialog.js

```diff
@@ -239,57 +238,54 @@ const CreateFilterDialog = ({
     };
 
     return (
-        <DialogContainer
+        <Dialog
+            fullWidth={true}
             open={open}
             onClose={handleClose}
             onKeyPress={handleKeyPressed}
         >
-            <CustomDialogTitle onClose={handleClose}>{title}</CustomDialogTitle>
-            <CustomDialogContent dividers>
-                <Grid container direction="row" spacing={1}>
-                    <Grid item xs={12} sm={8}>
-                        <TextField
-                            autoFocus
-                            margin="dense"
-                            type="text"
-                            style={{ width: '90%' }}
-                            onChange={(event) =>
-                                handleFilterNameChanges(event.target.value)
-                            }
-                            error={
-                                newNameList !== '' &&
-                                !filterNameValid &&
-                                !loadingCheckFilterName
-                            }
-                            label={inputLabelText}
-                        />
-                        {renderFilterNameStatus()}
-                        <RadioGroup
-                            aria-label="type"
-                            name="filterType"
-                            value={newListType}
-                            onChange={(e) => setNewListType(e.target.value)}
-                            row
-                        >
-                            <FormControlLabel
-                                value="SCRIPT"
-                                control={<Radio />}
-                                label={<FormattedMessage id="SCRIPT" />}
-                            />
-                            <FormControlLabel
-                                value="FORM"
-                                control={<Radio />}
-                                label={<FormattedMessage id="FORM" />}
-                            />
-                        </RadioGroup>
-                    </Grid>
-                </Grid>
+            <DialogTitle onClose={handleClose}>{title}</DialogTitle>
```

Comment:
if you don't use `DialogContainer`, `CustomDialogTitle`, `CustomDialogContent`  or `CustomDialogActions` anymore, remove it if not used elsewhere